sdks/go/pkg/beam/core/runtime/graphx/translate_test.go

```diff
@@ -157,8 +170,34 @@ func TestMarshal(t *testing.T) {
 			if got, want := len(p.GetRootTransformIds()), test.roots; got != want {
 				t.Errorf("got %d roots, want %d : %v", got, want, proto.MarshalTextString(p))
 			}
+			if got, want := p.GetRequirements(), test.requirements; !cmp.Equal(got, want) {
```

Comment:
Since order doesn't matter for the results, but in the future the SDK might add more than this requirement, consider adding a transformer to sort the slices for the equals check.

https://godoc.corp.google.com/pkg/google3/third_party/golang/cmp/cmp#example_Option_sortedSlice

Reply:
Done.